CONTRIBUTING.md

```diff
@@ -178,90 +190,83 @@ As languages differ quite a lot, here is a set of captures available to you when
-@constant.macro   ; constants defined by the preprocessor
+@comment               ; line and block comments
+@comment.documentation ; comments documenting code
 
-@module        ; modules or namespaces
-@string.special.symbol           ; symbols or atoms
-@string.special.url              ; URIs (e.g. hyperlinks)
-@string.special.path            ; filenames
+@comment.error         ; error-type comments (e.g., `DEPRECATED:`
+@comment.warning       ; warning-type comments (e.g., `WARNING:`, `TODO:`)
+@comment.note          ; note-type comments (e.g., `NOTE:`)
+@comment.info          ; info-type comments
 ```
 
 #### Markup
 
 Mainly for markup languages.
 
 ```query
-@markup.strong           ; bold text
+@markup.strong         ; bold text
 @markup.italic         ; text with emphasis
-@markup.underline        ; underlined text (only for literal underline markup!)
-@markup.strikethrough           ; strikethrough text
-@markup.heading            ; text that is part of a title
+@markup.strikethrough  ; strikethrough text
+@markup.underline      ; underlined text (only for literal underline markup!)
+
+@markup.heading        ; headings, titles (including markers)
+
 @markup.quote          ; block quotes
+@markup.math           ; math environments (e.g. `$ ... $` in LaTeX)
 @markup.environment    ; environments (e.g. in LaTeX)
-@markup.math             ; math environments (e.g. `$ ... $` in LaTeX)
+
 @markup.link           ; text references, footnotes, citations, etc.
 @markup.link.label     ; link, reference descriptions
 @markup.link.url       ; URL-style links
 
-@markup.raw          ; literal or verbatim text (e.g., inline code)
-@markup.raw.block    ; literal or verbatim text as a stand-alone block
+@markup.raw            ; literal or verbatim text (e.g., inline code)
+@markup.raw.block      ; literal or verbatim text as a stand-alone block
                        ; (use priority 90 for blocks with injections)
 
 @markup.list           ; list markers
 @markup.list.checked   ; checked todo-style list markers
 @markup.list.unchecked ; unchecked todo-style list markers
+```
 
-@comment.error         ; error-type comments (e.g., `DEPRECATED:`
-@comment.warning       ; warning-type comments (e.g., `WARNING:`, `TODO:`)
-@comment.note          ; note-type comments (e.g., `NOTE:`)
-@comment.info          ; info-type comments
-
-@diff.plus         ; added text (for diff files)
+```query
+@diff.plus       ; added text (for diff files)
 @diff.minus      ; deleted text (for diff files)
 @diff.delta      ; changed text (for diff files)
 ```
 
-#### Tags
-
-Used for XML-like tags.
-
 ```query
 @tag           ; XML tag names
 @tag.attribute ; XML tag attributes
 @tag.delimiter ; XML tag delimiters
 ```
 
-#### Conceal
+#### Non-highlighting captures
+
+```query
+@none    ; completely disable the highlight
+@error   ; syntax/parser errors
+```
+
+##### Conceal
 
 ```query
 @conceal ; for captures that are only used for concealing
```

Comment:
`@conceal` is not needed for concealing, only the `#set!` is needed. We can use conceal alongside something like markdown's list checked/unchecked markers to have both the highlight and the conceal. So should this PR also removes/deprecates the usage of this query?

Reply:
You're right; the capture is completely unnecessary -- if you _need_ a placeholder because no other capture fits, you can always use `@_conceal`.

Reply:
`@_conceal` still passed the highlights down...
I tested this query for `json`
```query
(escape_sequence) @string.escape
((escape_sequence) @_conceal
 (#eq? @_conceal "\\\"")
 (#set! conceal "\""))
```
And using `:Inspect` gives the result:
```
Treesitter                                                                                     
  - @string.json links to @string json                                                         
  - @string.escape.json links to @string.escape json                                           
  - @_conceal.json links to @_conceal json
```

I'm definitely surprised by this behavior, should this be documented in a later PR?